reviews/models.py

```diff
@@ -40,6 +40,19 @@ class Review(models.Model):
         (DISCONTINUED, _('Niet verder in behandeling genomen')),
     )
 
+    REVIEW_TYPES = [
+        ("supervisor", _("Beoordeling door eindverantwoordelijke")),
+        ("committee", _("Beoordeling door commissie")),
+    ]
```

Comment:
I'd like to see the type values extracted as class constants (like the stages are). That way you can refer to them like this:
``Review.TYPE_SUPERVISOR`` (name is not great, just for illustration purposes).